src/friendly-iframe-embed.js

```diff
@@ -629,3 +641,28 @@ export function whenContentIniLoad(elementOrAmpDoc, hostWin, rect) {
 export function isInFie(element) {
   return !!closestBySelector(element, '.i-amphtml-fie');
 }
+
+/**
+ * Preloads extensions. The extensions field is the preferred means as it
+ * supports versions.
+ * @param {?../extensions/amp-a4a/0.1/amp-ad-type-defs.CreativeMetaDataDef|!FriendlyIframeSpec} metaData
+ * @param {!Window} win
+ */
+export function preloadExtensions(metaData, win) {
+  if (metaData.extensions && metaData.extensions.length) {
```

Comment:
This "if one object type, do this, else if other type, do this" is icky. Why aren't we just standardizing a format to pass down?

Reply:
verified this is no longer used. Would've also helped if i read the comment here https://github.com/ampproject/amphtml/issues/7217#issuecomment-289822598, will remove customElementExtensions throughout codebase. Thanks